examples/petsc/area.c

```diff
@@ -136,8 +186,15 @@ int main(int argc, char **argv) {
   CHKERRQ(ierr);
   ierr = PetscOptionsEnd(); CHKERRQ(ierr);
 
+  // Setup function pointer for geometric factors
+  void (*geomfp)(void);
```

Comment:
I would declare this with the correct prototype and cast it on the line below, `(void(**)(void)&geomfp`, instead of needing the cast when creating a qfunction later.